HealthcareSystem/GraphicalEditor/Repository/MapObjectRepository.cs

```diff
@@ -0,0 +1,59 @@
+﻿using GraphicalEditor.Models;
+using GraphicalEditor.Repository.Intefrace;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphicalEditor.Repository
+{
+    public class MapObjectRepository : IMapObjectRepository
+    {
+        protected IEnumerable<MapObject> objects;
+        protected IRepository fileRepository;
+        protected MapObjectRepository() {
+            objects = fileRepository.LoadMap();
+        }
+
+        public MapObjectRepository(string path)
+        {
+        }
```

Comment:
Why is this empty?

Reply:
we will later use that path in next spring

Reply:
that is constructor too

Reply:
Okay, but why is it empty? Look at [this](https://stackoverflow.com/questions/2963742/empty-constructor-or-no-constructor#:~:text=You%20will%20have%20to%20add,the%20parameter%2Dless%20constructor%20alive.&text=The%20C%23%20compiler%20will%20create,one%2C%20with%20an%20empty%20body.) and inform me what you will use in future

Reply:
i will use it to open file to store data